src/app/(have-nav)/additionalDetail/page.tsx

```diff
@@ -0,0 +1,248 @@
+                className="block w-full rounded-md border-gray-300 pr-12 focus:border-blue-500 focus:ring-blue-500 sm:text-sm"
+              />
+              <div className="absolute inset-y-0 right-0 flex items-center pr-3">
+                <label htmlFor="floorSizeUnit" className="sr-only">
+                  Floor Size Unit
+                </label>
+                <select
+                  id="floorSizeUnit"
+                  className="h-full rounded-md border-transparent bg-transparent py-0 pl-2 pr-7 text-gray-500 focus:border-blue-500 focus:ring-blue-500 sm:text-sm"
+                >
+                  <option value="sqm">sqm</option>
+                  <option value="sqft">sqft</option>
+                  {/* Add other units if needed */}
+                </select>
+              </div>
+            </div>
+          </div>
+          <div>
+            <label
+              htmlFor="unitNumber"
+              className="text-md block font-medium text-gray-700"
+            >
+              Unit Number
+            </label>
+            <input
+              type="number"
+              id="unitNumber"
+              value={unitNumber}
+              onChange={(e) => setUnitNumber(e.target.value)}
+              className="mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-blue-500 focus:ring-blue-500 sm:text-sm"
+            />
+          </div>
+        </div>
+      </div>
+      <div className="mb-4">
+        <h3 className="text-md mb-2">Upload Photos of Your Property</h3>
+        <div className="flex w-full items-center justify-center">
+          <label
+            htmlFor="photo-upload"
+            className="flex h-48 w-full cursor-pointer flex-col items-center justify-center rounded-md bg-ci-gray outline outline-2 outline-ci-dark-gray hover:bg-ci-dark-gray"
+          >
+            <svg
+              className="mb-2 h-8 w-8 text-gray-400"
+              fill="none"
+              stroke="currentColor"
+              viewBox="0 0 24 24"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                strokeWidth={2}
+                d="M4 16l4.586-4.586a2 2 0 012.828 0L16 16m-2-2l1.586-1.586a2 2 0 012.828 0L20 14m-6-6h.01M6 20h12a2 2 0 002-2V6a2 2 0 00-2-2H6a2 2 0 00-2 2v12a2 2 0 002 2z"
+              />
+            </svg>
+            <span className="text-sm text-gray-500">Add photos</span>
+            <input
+              id="photo-upload"
+              type="file"
+              className="hidden"
+              multiple
+              onChange={handlePhotoUpload}
+            />
+          </label>
+        </div>
+        <div className="mt-4 flex w-full overflow-x-auto">
+          {photos.map((photo, index) => (
+            // w-44
+            <div key={index} className="relative mb-2 mr-2 h-28 w-64">
+              <img
```

Comment:
- Could you use `next/image` instead of `img` because it's [Next.js native tag](https://nextjs.org/docs/pages/api-reference/components/image).
- I haven't tried it yet it might or might not work. You can try set the outer div `overflow-x-scroll` and fixes inner `Image` size. That might make it horizontally scrollable.